client/systray/systray.go

```diff
@@ -295,6 +296,14 @@ func (menu *Menu) rebuild() {
 		})
 	}
 
+	// TODO(#15528): this menu item shouldn't be necessary at all,
+	// but is at least more discoverable than having users switch profiles or exit nodes.
+	menu.rebuildMenu = systray.AddMenuItem("Rebuild menu", "Fix missing menu items")
+	onClick(ctx, menu.rebuildMenu, func(_ context.Context) {
+		menu.rebuildCh <- struct{}{}
```

Comment:
not sure if it matters much, but:
```suggestion
	onClick(ctx, menu.rebuildMenu, func(ctx context.Context) {
		select {
		case <-ctx.Done():
		case menu.rebuildCh <- struct{}{}:
		}
```